deepchem/hyper/tests/test_grid_hyperparam_opt.py

```diff
@@ -0,0 +1,157 @@
+"""
+Tests for hyperparam optimization.
+"""
+import os
+import unittest
+import tempfile
+import shutil
+import numpy as np
+import tensorflow as tf
+import deepchem as dc
+import sklearn
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.ensemble import RandomForestRegressor
+
+
+class TestGridHyperparamOpt(unittest.TestCase):
```

Comment:
I think it is better to add assertion about valid_score like GPGO test in all tests.

```
assert valid_score["your_metric"] == max(all_results.values())
```

Reply:
Added in!